wls-gui-wahllokalsystem/src/views/auszaehlung/ErfassungStimmzettelView.vue

```diff
@@ -23,23 +23,12 @@ const { isUWB } = storeToRefs(useUserStore());
 const { wahlenActions } = useWahlenStore();
 const { getStimmzettelTermForWahl } = useTextFormatter();
 
-const wahlID = computed(() => route.params.wahlId as string);
-const wahl = computed(() => {
-  if (wahlID.value) {
-    return wahlenActions.getWahlOrUndefinedById(wahlID.value);
-  } else {
-    return undefined;
-  }
-});
+const wahlID = route.params.wahlId as string;
+const wahl = wahlID ? wahlenActions.getWahlOrUndefinedById(wahlID) : undefined;
```

Comment:
```suggestion
const wahl = wahlenActions.getWahlOrUndefinedById(wahlID);
```

Bei den anderen Verwendungen wird das Handling des Parameters der Function überlassen.